test/WebJobs.Script.Tests.Integration/TestScripts/Node/HttpTriggerToBlob/function.json

```diff
@@ -10,7 +10,7 @@
             "type": "blob",
             "name": "outBlob",
             "direction": "out",
-            "path": "test-output-node/{headers.prefix}-{id}-{query.suffix}"
+            "path": "test-output-node/{headers.prefix}-{id}-{query.suffix}-{metadata.m2}"
```

Comment:
This binding to a nested body object is what wasn't working before - only top level simple properties were supported.